src/lib/progress.js

```diff
@@ -26,5 +30,12 @@ export function createProgressBar() {
     el.classList.toggle("finished", completed === total);
     el.style.setProperty("--progress-total", total);
     el.style.setProperty("--progress-completed", completed);
+    const status = browser.i18n.getMessage("pickerProgress", [
+      Math.floor(completed * 100 / total),
+      completed,
+      total,
+      failed
+    ]);
+    document.title = status;
```

Comment:
I found that it is better to put the status on title instead of the progress bar. The progress bar is too thin.